components/payloads/cu_ros_payloads/src/std_msgs.rs

```diff
@@ -0,0 +1,25 @@
+use serde::{Deserialize, Serialize};
+
+use crate::RosMsgAdapter;
+
+// std_msgs/Int8 like struct
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Int8 {
+    pub data: i8,
+}
```

Comment:
maybe Int8(i8)?

Reply:
It should work but I think it is better to stick to ROS implementation https://docs.ros.org/en/melodic/api/std_msgs/html/msg/Int8.html